airbnb.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Airbnb",
-  "description": "Search and retrieve information about Airbnb listings",
+  "description": "# Airbnb MCP Server\n\nMCP Server for searching Airbnb and get listing details.\n\n## Features\n\n- Respects Airbnb's robots.txt rules\n- Uses cheerio for HTML parsing\n- No API key required\n- Returns structured JSON data\n- Reduces context load by flattening and picking data\n\n## Tools\n\n1. `airbnb_search`\n   - Search for Airbnb listings\n   - Required Input: `location` (string)\n   - Optional Inputs:\n     - `placeId` (string)\n     - `checkin` (string, YYYY-MM-DD)\n     - `checkout` (string, YYYY-MM-DD)\n     - `adults` (number)\n     - `children` (number)\n     - `infants` (number)\n     - `pets` (number)\n     - `minPrice` (number)\n     - `maxPrice` (number)\n     - `cursor` (string)\n     - `ignoreRobotsText` (boolean)\n   - Returns: Array of listings with details like name, price, location, etc.\n\n2. `airbnb_listing_details`\n   - Get detailed information about a specific Airbnb listing\n   - Required Input: `id` (string)\n   - Optional Inputs:\n     - `checkin` (string, YYYY-MM-DD)\n     - `checkout` (string, YYYY-MM-DD)\n     - `adults` (number)\n     - `children` (number)\n     - `infants` (number)\n     - `pets` (number)\n     - `ignoreRobotsText` (boolean)\n   - Returns: Detailed listing information including description, host details, amenities, pricing, etc.\n\n## License\n\nThis MCP server is licensed under the MIT License.\n\n## Disclaimer\n\nAirbnb is a trademark of Airbnb, Inc.\nOpenBnB is not related to Airbnb, Inc. or its subsidiaries\n",
```

Comment:
I want to start supporting consuming yaml manifests so that we can use multiline strings here.